boms/user/server/expansion/standard/pom.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!--
+  ~ Copyright The WildFly Authors
+  ~ SPDX-License-Identifier: Apache-2.0
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.wildfly.bom.builder</groupId>
+        <artifactId>wildfly-server-expansion</artifactId>
+        <!--
+        Maintain separation between the artifact id and the version to help prevent
+        merge conflicts between commits changing the GA and those changing the V.
+        -->
+        <version>34.0.0.Beta1-SNAPSHOT</version>
+    </parent>
+
+    <packaging>pom</packaging>
+
+    <artifactId>wildfly-server-expansion-standard-builder</artifactId>
+
+    <name>WildFly BOM Builders: Server Expansion Standard Builder</name>
+    <description>This artifact builds a bill of materials (BOM) for Server Expansion Standard's Dependency Management</description>
+
+    <dependencyManagement>
+        <dependencies>
+            <!-- base import of server's dep management -->
+            <dependency>
+                <groupId>org.wildfly</groupId>
+                <artifactId>wildfly-standard-expansion-bom</artifactId>
+                <version>${full.maven.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+
+    <!-- enforces all BOM artifacts are available -->
+    <dependencies>
+        <dependency>
+            <groupId>org.wildfly</groupId>
+            <artifactId>wildfly-galleon-pack</artifactId>
+            <type>pom</type>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.wildfly.plugins</groupId>
+                <artifactId>wildfly-bom-builder-plugin</artifactId>
+                <inherited>false</inherited>
+                <executions>
+                    <execution>
+                        <id>build-bom</id>
+                        <goals>
+                            <goal>build-bom</goal>
+                        </goals>
+                        <configuration>
+                            <parent>
+                                <groupId>${bomGroupId}</groupId>
+                                <artifactId>${bomArtifactIdPrefix}-ee</artifactId>
+                                <version>${project.version}</version>
+                                <relativePath/>
+                            </parent>
```

Comment:
I would prefer to defer this change. It seems independent from the rest and is something that can come later without breaking users. OTOH it hasn't been discussed much and once we do it this way it's an incompatible change to go back to having this be independent.

Reply:
reverted

Reply:
please note that by reverting this change I added back the EE platform artifacts required by MicroProfile, so dep management content matches the MicroProfile BOM at main